docs/src/content/how-to/web-workers.raw.md

```diff
@@ -0,0 +1,57 @@
+## Web Workers
+
+Web Workers are a tool run scripts in background threads. The worker thread can perform tasks without interfering with the user interface. This can be used to offload heavy tasks from the main js thread. Extra information about this subject can be found on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers)
```

Comment:
```suggestion
Web Workers are a tool to run scripts in background threads. The worker threads can perform tasks without interfering with the user interface. This can be used to offload heavy tasks from the main js thread. Extra information about this subject can be found on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers)
```